cram_core/cram_executive/src/perform.lisp

```diff
@@ -179,6 +179,7 @@ similar to what we have for locations.")
 
 (defun log-action-parameter (designator action-id)
   (cond ((desig-prop-value designator :effort) (ccl::send-effort-action-parameter action-id (write-to-string (desig-prop-value designator :effort))))
+        ((desig-prop-value designator :position) (ccl::send-position-action-parameter action-id (desig-prop-value designator :position)))
```

Comment:
There is no "position" field in the designators...
Don't see how this can work.